tests/ui/const-generics/generic_const_parameter_types/forward_declared_type.rs

```diff
@@ -0,0 +1,11 @@
+#![feature(adt_const_params, generic_const_parameter_types)]
+#![expect(incomplete_features)]
+
+use std::marker::PhantomData;
+
+struct UsesConst<const N: [u8; M], const M: usize>;
```

Comment:
that's a bad error, pls add FIXME

Reply:
added it to the nameres code instead of the ui test since I usually forget to change my vscode filter to nothing instead of `compiler/` when i grep for FIXME's :ferrisClueless: